tests/install-deps.sh

```diff
@@ -2,6 +2,16 @@
 
 # Helper script for Continuous Integration Services
 
+# Check if the last commit message requests the CI to skip the build.
+git log HEAD~..HEAD > last-commit
```

Comment:
Indeed, I would suggest not to add `[skip ci]` in any commits of a pull request if the CI is not intended to be skipped.